katello/katello-backup

```diff
@@ -1,94 +1,113 @@
 #!/bin/bash
 
 usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
+  echo "Usage: katello-backup --directory <backup directory>"
+  echo " eg: $ katello-backup --directory /tmp/katello-backup/"
```

Comment:
I'm happy to go to a positional argument, I had asked for feedback on argument vs option and didn't really hear anything back so just went with options.